src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -663,9 +663,9 @@ private static SyntaxTokenList UpdateDeclarationAccessibility(SyntaxTokenList mo
 
             // TODO: Move more APIs to use pooled ArrayBuilder
```

Comment:
Is this issue addressed by this PR? Or we need to keep it open for similar problems somewhere else?

Reply:
can likely be removed.  will do in future pr.